libsolidity/codegen/ir/Common.cpp

```diff
@@ -24,6 +25,21 @@ using namespace solidity;
 using namespace solidity::util;
 using namespace solidity::frontend;
 
+Arity frontend::getFunctionArity(FunctionDefinition const& _function)
```

Comment:
Usually we try to avoid `get` prefixes.

Reply:
Yeah, I saw that. The reason I added a prefix is that `functionArity` or `functionName` is a very natural choice for a variable name. It's less of a problem if the function is namespaced inside a class but a free function will clash a lot. I think it's better to have a prefix than to make naming local variables harder.

Unfortunately I could not come up with a prefix better than `get`. Maybe you have a good suggestion?

Reply:
Yes: Avoid free functions :)

Reply:
Even if they would be just static functions when put in a class?

Or maybe I should create a namespace for them?

Reply:
Done in #8949.

I converted then into factory methods in the `Arity` struct.